blog_post_collaboration/Integrating_Kedro_with_Databricks.md

```diff
@@ -0,0 +1,136 @@
+# The ways of integrating Kedro with Databricks
+ 
+In recent months we've updated Kedro documentation to illustrate [three different ways of integrating Kedro with Databricks](https://docs.kedro.org/en/stable/deployment/databricks/index.html). 
+
+* You can choose a [workflow based on Databricks jobs](https://docs.kedro.org/en/stable/deployment/databricks/databricks_deployment_workflow.html) to deploy a project that finished development
+* For faster iteration on changes, the workflow documented in ["Use a Databricks workspace to develop a Kedro project"](https://docs.kedro.org/en/stable/deployment/databricks/databricks_notebooks_development_workflow.html) is for those who prefer to develop and test their projects directly within Databricks notebooks, to avoid the overhead of setting up and syncing a local development environment with Databricks. 
+* Alternatively, you can work locally in an IDE as described by the workflow documented in ["Use an IDE, dbx and Databricks Repos to develop a Kedro project"](https://docs.kedro.org/en/stable/deployment/databricks/databricks_ide_development_workflow.html). This is ideal if you’re in the early stages of learning Kedro, or your project requires constant testing and adjustments since you can use your IDE’s capabilities for faster, error-free development, while testing on Databricks. However, the experience is still not ideal: you must sync your work inside Databricks with dbx and run the pipeline inside a notebook. Debugging has a long setup for each change and there is less flexibility than inside an IDE. 

```

Comment:
I think it is too optimistic here. It's weird if this workflow is always better and we suggest something else otherwise in Docs. IIRC, this works nicely if you have a pure spark workflow. This should be the most important question if one want to decide to invest in databricksConnect workflow

Reply:
Don't worry too much about the phrasing as I can smooth that part when I transfer to the repo. Diego is part of a QB team so it's OK to say "we" as long as we are clear who that "we" is, and we can explain that this workflow is one of several options for working with databricks.